solarwinds_apm/version.py

```diff
@@ -1 +1 @@
-__version__ = "3.5.0"
+__version__ = "4.0.0.0"
```

Comment:
Is it a typo?

Reply:
Nope, for staging/test publishes we use a 4th digit.